### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by lpham598**
Set up database. Set up backend and express. Fetched data from plant API.

### CODE REVIEW

**Comment by fansfansfansfans** [comment]
fantastic Lia!

File: server/controllers/plantController.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                if(species.growth.precipitation_minimum.inches){
+                    //console.log(species.growth.precipitation_minimum.inches)
+                    resultObj.precipitation_minimum = species.growth.precipitation_minimum.inches;
+                }
+                if(species.growth.resprout_ability){
+                    //console.log(species.growth.resprout_ability)
+                    resultObj.resprout_ability = species.growth.resprout_ability;
+                }
+                if(species.growth.shade_tolerance){
+                    //console.log(species.growth.shade_tolerance)
```
Comment (simple-sifu) [inline server/controllers/plantController.js:92]: @lia can you remove these console.logs?

File: server/controllers/plantController.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+                }
+                Plant.create(resultObj);
+		    }else{
+			    console.log("response not ok")
+            }
+            
+        }  
+    }
+
+  //console.log('**********this are plants: ', plants, 'plant-length: ', plants.length);
```
Comment (simple-sifu) [inline server/controllers/plantController.js:131]: Pls remove these also

File: server/controllers/plantController.js (hunk 1 above)
Comment (lpham598) [reply server/controllers/plantController.js]: It's been removed.

Comment (simple-sifu) [reply server/controllers/plantController.js]: Thanks Lia

**Pull request merged**